### PR TITLE
Change trigger that run test

### PR DESCRIPTION
**Pull request by arukuka**
# Summary

Same as https://github.com/acompany-develop/QuickMPC/pull/31

Update workflow in githubactions

# Purpose
https://www.bitsnbites.eu/a-stable-mainline-branching-model-for-git/
In adopting this branching strategy, the role of CI was changed

# Contents

test.yml: ensuring quality through testing
```
on:
  pull_request:
  push:
    branches:
      - main
      - "release/**"
```

# Testing Methods Performed
by this ci

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -2,9 +2,11 @@ name: Python CI
 
 on:
   pull_request:
+  push:
     branches:
-      - develop
-      - migration/**
+      - main
+      - "release/**"
+      - "feature/**"
```
Comment (mdonaka) [inline .github/workflows/test.yml]: I want `feature/**` to run only when pull_request.
https://github.com/acompany-develop/QuickMPC/pull/31#discussion_r964642482

Comment (arukuka) [reply .github/workflows/test.yml]: @mdonaka noted with thanks.  I think so.

Comment (arukuka) [reply .github/workflows/test.yml]: @mdonaka I remote it.  please check :bow:

Comment (mdonaka) [reply .github/workflows/test.yml]: I checked it. Thank you!

**Review by mdonaka — APPROVED** [review]

**Review by come2ry — APPROVED** [review]

**Pull request merged**